packages/logger/src/types/index.ts

```diff
@@ -10,3 +10,9 @@ export type {
   LoggerInterface,
 } from './Logger.js';
 export type { UnformattedAttributes } from './logKeys.js';
+export type {
+  JSONPrimitive,
+  JSONObject,
+  JSONArray,
+  JSONValue,
+} from './json.js';
```

Comment:
File doesn't exist, and we should not re-export types we already have in `@aws-lambda-powertools/commons/types`.